src/sentry/static/sentry/app/views/settings/projectTags.jsx

```diff
@@ -127,3 +123,19 @@ export default class ProjectTags extends AsyncView {
     );
   }
 }
+
+const TagPanelItem = styled(PanelItem)`
+  padding: 0;
+  align-items: center;
+`;
+
+const TagName = styled('div')`
+  flex: 1;
+  padding: ${space(2)};
```

Comment:
Was `flex-end` not included here because it was unnecessary?

Reply:
Yeah, there was/is a lot of unnecessary CSS throughout :|